pkg/controller/component/log.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 Vesoft Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package component
+
+import (
+	"github.com/go-logr/logr"
+
+	"github.com/vesoft-inc/nebula-operator/pkg/logging"
+)
+
+var _log = logging.Log.WithName("controller")
+
+func getLog() logr.Logger { return _log }
```

Comment:
The function getLog()  has  some duplicates,  why not use 'log' directly?

Reply:
I was use `log` directly. 
But I found that the log is easy to be overwritten while using. Therefore, the global `log` variable has changed.

for example:

```golang
func example() {
    log = log.WithValues(...) // : missed
}
```